diff --git a/src/Makefile.cfg b/src/Makefile.cfg
index 075cd2d3a..75b910c86 100644
--- a/src/Makefile.cfg
+++ b/src/Makefile.cfg
@@ -61,10 +61,7 @@ ifeq   (,$(filter GCC% CLEANONLY,$(.VARIABLES)))
 
   # If this version is not in the list, default to the latest supported
   ifeq (,$(filter $(v),$(SUPPORTED_GCC_VERSIONS)))
-	define line =
-	Your compiler version, GCC $(version), is not supported by the Makefile.
-	The Makefile will assume GCC $(LATEST_GCC_VERSION).))
-	endef
+	line = Your compiler version, GCC $(version), is not supported by the Makefile. The Makefile will assume GCC $(LATEST_GCC_VERSION).
    $(call print,$(line))
    GCC$(subst .,,$(LATEST_GCC_VERSION))=1
   else
